tests/Vogen.Tests/InstanceGenerationTests.cs

```diff
@@ -280,14 +280,20 @@ public void It_generates_a_valid_item_when_given_a_valid_DateTime_input()
             [Fact]
             public void It_generates_a_valid_item_when_given_a_valid_DateTimeOffset_input()
             {
+                var timezoneOffset = TimeZoneInfo.Local.BaseUtcOffset;
```

Comment:
Works great on my machine - thank you so much - it's great to be able to get these passing everywhere!